schema/tracer_provider.json

```diff
@@ -89,6 +92,28 @@
             "type": ["object", "null"],
             "additionalProperties": false
         },
+        "ExperimentalComposableSampler": {
+            "type": "object",
+            "additionalProperties": {
+                "type": ["object", "null"]
+            },
+            "minProperties": 1,
+            "maxProperties": 1,
+            "properties": {
+                "always_off": {
+                    "$ref": "#/$defs/AlwaysOffSampler"
```

Comment:
Consistency with ExperimentalComposableParentBasedSampler would suggest separate ExperimentalComposableAlwaysOn/ExperimentalComposableAlwaysOff sampler as defined in the spec, but it seems silly at the same time.